.babelrc

```diff
@@ -1,4 +1,19 @@
 {
-  "presets": ["env", "react", "stage-1"],
-  "plugins": ["add-module-exports"]
+  "env": {
+    "development": {
+      "presets": ["env", "react", "stage-1"],
```

Comment:
One question just popped my mind: do we still need `stage-1` when using `env` preset? If so, what for?

Reply:
I have never used that preset. The documentation of env says:
"Without any configuration options, babel-preset-env behaves exactly the same as babel-preset-latest (or babel-preset-es2015, babel-preset-es2016, and babel-preset-es2017 together)."

I think that the stage-1 is only needed, if there is some code that depend on current stage-1 propositions.

Reply:
Yeah, I think stage-1 was needed for rest-spread. I'm wondering how env will transpile it, though.

Reply:
env preset does not yet transpile object-rest-spread. I am using the following:
```
plugins: ["transform-object-rest-spread"]
```
I think that makes it more definitive.